db/migrate/1733318438269_add_new_fields_to_users.rb

```diff
@@ -0,0 +1,19 @@
+class AddNewFieldsToUsers < ActiveRecord::Migration[6.0]
+  def up
+    add_column :users, :subscription_plan, :string, null: false
+    add_column :users, :image, :string, null: false
+    add_column :users, :verified, :boolean, default: false
+    add_column :users, :login_count, :integer, default: 0
+    add_column :users, :last_login_at, :datetime
+    add_column :users, :password, :string, null: false
```

Comment:
The password should not be stored as plain text in the database. Consider using a secure method to hash passwords, such as bcrypt, to enhance security.